tests/integration/post_processors/test_otel_live_export.py

```
@@ -0,0 +1,113 @@
+    """Minimal OTLP HTTP sink that records POST /v1/metrics timestamps."""
+
+    def do_POST(self) -> None:  # noqa: N802
+        if self.path == "/v1/metrics":
+            content_length = int(self.headers.get("Content-Length", 0))
+            self.rfile.read(content_length)
+            self.server.received_exports.append(True)  # type: ignore[attr-defined]
+        self.send_response(200)
+        self.end_headers()
+        self.wfile.write(b"{}")
+
+    def log_message(self, format: str, *args: object) -> None:
```

Ruff (0.15.12)

[error] Function argument `format` is shadowing a Python builtin

(A002)